xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -377,6 +377,18 @@ public String loaderWriter() {
             return configClass;
           }
 
+          @Override
+          public String resilienceStrategy() {
+            String resilienceClass = null;
+            for (BaseCacheType source : sources) {
+              resilienceClass = source.getResilience();
+              if (resilienceClass != null) {
+                return resilienceClass;
+              }
+            }
+            return resilienceClass;
```

Comment:
I would prefer `return null` here and `String resilienceClass` in the loop. Less cryptic

Reply:
This is stylistically consistent with the rest of this class.  Would you prefer me to change just this method, or try and change all of them?

Reply:
Ok. Keep it that way... We can PR later on the full class